compatibility-kit/javascript/features/parameter-types/parameter-types.ts

```diff
@@ -21,3 +21,8 @@ Given('{flight} has been delayed {int} minutes', function(
   assert.equal(flight.to, 'CDG')
   assert.equal(delay, 45)
 })
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+Given('{airport} is closed because of a strike', function(airport) {
```

Comment:
I know we have the {x} test in rspec for ruby, but do these tests need copying elsewhere, or does the CCK function like the golden tests?

Reply:
The CCK defines golden `ndjson` files by running the features against `fake-cucumber`. We don't require other Cucumber implementations to produce identical files, but the files they produce should be consumable by the `json-formatter` and `html-formatter`.